packages/react/src/components/svg/SvgNames.ts

```diff
@@ -458,6 +458,7 @@ export const SVG_NAMES = {
     youtubeSourceSm: 'youtubeSourceSm',
     zoomNegative: 'zoomNegative',
     zoomPositive: 'zoomPositive',
+    githubMark: 'githubMark',
```

Comment:
Should it maybe just be `github`?

Reply:
I tried to follow the naming here https://github.com/logos

Reply:
Makes sense :+1:  Thanks